src/Modules/ContentBlocks/module.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module>
+  <name>ContentBlocks</name>
+  <version>5.0.0</version>
+  <requirements>
+    <minimum_version>6.0.0</minimum_version>
+    <maximum_version>7.0.0</maximum_version>
+  </requirements>
+  <authors>
+    <author>
+      <name><![CDATA[Davy Hellemans]]></name>
+      <url><![CDATA[http://blog.bauffman.be]]></url>
+    </author>
+    <author>
+      <name><![CDATA[Tijs Verkoyen]]></name>
+      <url><![CDATA[https://www.sumocoders.be]]></url>
+    </author>
+    <author>
+      <name><![CDATA[Matthias Mullie]]></name>
+      <url><![CDATA[http://www.mullie.eu]]></url>
+    </author>
+    <author>
+      <name><![CDATA[Jelmer Prins]]></name>
+      <url><![CDATA[https://www.sumocoders.be]]></url>
+    </author>
+    <author>
+      <name><![CDATA[Jeroen Desloovere]]></name>
```

Comment:
Remove this. Jeroen is not an author of the ContentBlocks

Reply:
removed in https://github.com/forkcms/forkcms/pull/3555/files#diff-904113c4398a065037a842026ffc8cce600daf8ea5bf3bc8d07545cf9fc8db1f